Auth0/SignupPasskey.swift

```diff
@@ -5,31 +5,8 @@ import AuthenticationServices
 /// The signup passkey credential obtained from the [`ASAuthorizationControllerDelegate`](https://developer.apple.com/documentation/authenticationservices/asauthorizationcontrollerdelegate) delegate.
 /// Contains the subset of relevant properties from [`ASAuthorizationPlatformPublicKeyCredentialRegistration`](https://developer.apple.com/documentation/authenticationservices/asauthorizationplatformpublickeycredentialregistration).
 @available(iOS 16.6, macOS 13.5, visionOS 1.0, *)
-public protocol SignupPasskey {
-
-    var credentialID: Data { get }
-    var attachment: ASAuthorizationPublicKeyCredentialAttachment { get }
-    var rawClientDataJSON: Data { get }
-    var rawAttestationObject: Data? { get }
-
-}
+public protocol SignupPasskey: NewPasskey {}
 
 @available(iOS 16.6, macOS 13.5, visionOS 1.0, *)
 extension ASAuthorizationPlatformPublicKeyCredentialRegistration: SignupPasskey {}
-
-@available(iOS 16.6, macOS 13.5, visionOS 1.0, *)
-extension ASAuthorizationPublicKeyCredentialAttachment {
```

Comment:
This was moved into `NewPasskey.swift`, so it can be used by both the Authentication API client and the My Account API client.